wp-content/themes/oceanwp-child/assets/css/template-styles.css

```diff
@@ -1,5 +1,7 @@
 p{
     font-size:16px !important;
+    line-height: 150%;
+    font-family:'Lato';
 }
 
 h1{
```

Comment:
Please notice we have a full typography setup in the design - so just copy it to our template css.
Here's how H1 looks like:
<img width="166" alt="image" src="https://user-images.githubusercontent.com/6459899/155829985-bef33268-f239-4a32-96bd-75c8b36a34d9.png">
You can get it [here](https://www.figma.com/file/vEG8pWuftMli0PwYMeQj2j/GGRC-design?node-id=2483%3A24550)